src/services/completions.ts

```diff
@@ -3124,6 +3125,15 @@ function getContextualType(previousToken: Node, position: number, sourceFile: So
     }
 }
 
+/**
+ * An accessible symbol is a local in the same block or any enclosing blocks, up to and including the global scope (note imports are global). 

```

Comment:
I don't know if this comment is 100% accurate, but this is my understanding of what this method does. Open to change this if it's wrong.